### ISSUE TITLE
Reduce traces on info level

### ISSUE THREAD

**Issue by markheger** [IBMStreams/streamsx.eventstore#10]
currently many traces appear on info level, 
try to increase level to debug/trace

**Comment by markheger** [comment]
At INFO level there are many traces from io.netty:

```
14 Mar 2019 11:09:07.496+0100 [25920] INFO #splapptrc,J[1],P[1],InsertSampleComp_count M[Slf4JLogger.java:io.netty.util.internal.logging.Slf4JLogger.info:101]  - [id: 0xc701b30c] REGISTERED
14 Mar 2019 11:09:07.497+0100 [25920] INFO #splapptrc,J[1],P[1],InsertSampleComp_count M[Slf4JLogger.java:io.netty.util.internal.logging.Slf4JLogger.info:101]  - [id: 0xc701b30c] REGISTERED

```

Try to switch off logs from this module.

**Comment by markheger** [comment]
resolved with 2.0
